backend/src/openbeheer/health_checks/checks.py

```diff
@@ -130,11 +133,9 @@ def run(self) -> HealthCheckResult:
                         severity="warning",
                     )
                 )
+        else:
+            return HealthCheckResult(check=self, errors=errors)
 
-        if catalogue_found:
-            return HealthCheckResult(
+        return HealthCheckResult(
```

Comment:
Now that the default for errors is [],
this is not a special case any more. We can just always return `HealthCheckResult(check=self, errors=errors)`